src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -24,7 +27,7 @@ import createLogger from "Utils/logger"
 import { Media } from "Utils/Responsive"
 import { offerFlowSteps, OrderStepper } from "../../Components/OrderStepper"
 
-export interface OfferProps {
+export interface OfferProps extends ErrorModalProps {
```

Comment:
This line caught my attention and made me wonder whether Context is the correct thing to use for building an error modal component that can be used by any step in the Make Offer flow.

The Offer component should only care about being able to show an error modal and tell it what to display. However, this component is now setup as a consumer of the error modal's Context to populate its props that inherit from the error modal's props, which seems more complex than it needs to be.

Reply:
Good point! I'll change it back to only caring about the one `showErrorModal` prop.